client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -96,6 +114,18 @@ export const useRequestColumns = () => {
       width: 150,
       getSortValue: rowData => rowData.requestedQuantity,
     },
+    {
+      key: 'requestedNumPacks',
+      label: 'label.requested-packs',
+      description: 'label.requested-number-packs',
+      align: ColumnAlign.Right,
+      // width: 150,
+      accessor: ({ rowData }) =>
+        formatNumber.roundUp(
```

Comment:
can you change to `NumUtils.round` with 2dp please. the rounding up isn't accurate and I'm concerned that we're giving the wrong impression. 
the supplying store will likely only send whole packs, but then the requesting store will likely only ever order multiples of pack sizes

Reply:
Sure, Richard did actually specify rounding up to the nearest whole number in the issue, but I've changed it to 2.dp. I used `formatNumber.round()`, since it was already in use in this file -- looks like we have some duplication of utility methods :)

2e299d8cc